pkg/stanza/fileconsumer/reader.go

```diff
@@ -178,8 +179,8 @@ func (r *Reader) Read(dst []byte) (int, error) {
 	}
 	n, err := r.file.Read(dst)
 	appendCount := min0(n, r.fingerprintSize-int(r.Offset))
-	// return for n == 0 or r.Offset >= r.fileInput.fingerprintSize
 	if appendCount == 0 {
+		r.Debugw("Fingerprint does not need to be updated")
```

Comment:
This is likely to create excessive logging, even at the debug level in my opinion. Is it really valuable?

Reply:
It is not needed